### PR TITLE
[DEV-1048] Add AccountActivated page

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes

Add `auth/account-activated` route and page.

#### Motivation and Context

The page is the last step of the registration flow

#### How Has This Been Tested?

Manually

#### Screenshots (if appropriate):
Desktop:
image

Mobile:
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/atoms/IconFireworks/IconFireworks.tsx
Comment (kin0992) [inline apps/nextjs-website/src/components/atoms/IconFireworks/IconFireworks.tsx:1]: @marcobottaro check the 🧵 here: https://github.com/pagopa/developer-portal/pull/356#discussion_r1377843397

**Review by kin0992 — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/organisms/Auth/AccountActivatedCard.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <Typography variant='body1' mb={2} textAlign='center'>
+              {accountActivated.welcomeMessage}
+            </Typography>
+            <Stack spacing={4} pt={4} pb={4}>
+              <Stack direction='row' justifyContent='center'>
+                <Link href='/'>
+                  <Button variant='contained'>
+                    {accountActivated.goToDashboard}
+                  </Button>
+                </Link>
```
Comment (kin0992) [inline apps/nextjs-website/src/components/organisms/Auth/AccountActivatedCard.tsx:36]: Just a curiosity: why a `Button` within a `Link` and not 
```
<Button variant='contained' component={Link} href='/'>
    {accountActivated.goToDashboard}
</Button>
```
is there any difference between these two snippets?

Comment (marcobottaro) [reply apps/nextjs-website/src/components/organisms/Auth/AccountActivatedCard.tsx:36]: I don't think there's a difference, but your solution is more elegant

**Review by kin0992 — APPROVED** [review]
💪

**Review by datalek — APPROVED** [review]

**Pull request merged**